chapters/part3/hyperparameter-optimization-with-optuna.md

```diff
@@ -0,0 +1,363 @@
+---
+title: 'Hyperparameter Optimization with Optuna'
+description: "This chapter gives a basic tutorial for optimizing hyperparameters of your model."
+type: chapter
+---
+
+<textblock>
+
+In this chapter, we'll give a quick tutorial of hyperparameter optimization for your AllenNLP model.
+
+</textblock>
+
+<exercise id="1" title="Hyperparameter matters!">
+
+A choice of hyperparameters sometimes has a strong impact to the performance of a model.
+Even if you use the same model, performance could drastically change by the hyperparameters (e.g. learning rate, dimensionality of word embeddings) you use.
+Following figure shows the performance differences with changing hyperparameters.
+<img src="/part3/hyperparameter-optimization-with-optuna/hyperparameter_matters.jpg" alt="Hyperparameter matters!" />
+
+The choice of hyperparameters is often based on heuristics of experts, or random search, and may complicate the code.
```

Comment:
```suggestion
The choice of hyperparameters is often based on recommended values or random search and may complicate the code.
```

Reply:
Hmm, I'm wondering `recommended values` would not express my intension.
I want to say that `hyperparameters are often selected based on human experiences, without any evidence`.

Reply:
This is not a strong opinion, but IMO, `recommended values` would be aligned with the following explanation in section 2:

> These hyperparameters are selected based on the standard recommendations.

Reply:
I tried to update here with explaining a typical story of hyperparameter optimization 8e6997a.
What do you think about it?